app/decorators/fa1_interview_pdf_decorator.rb

```diff
@@ -4,6 +4,7 @@ def attributes
       fa_name: fee_agent_name,
       fa_email: fee_agent_email,
       fa_number: fee_agent_phone_number,
+      interview_date: created_at.strftime("%m/%d/%Y"),
```

Comment:
I think we'll want to make sure this date is in the right timezone for the state (we do it like this for Colorado: https://github.com/codeforamerica/colorado-benefits/blob/3e1ef790960bfa6f9fea439b0b80a898d36ee5ce/app/decorators/report_decorator.rb#L122

Reply:
Good idea. I'll do it in the morning.